data/chameleoncloud/sites/nrp/clusters/chameleon/nodes/ef4f8c1c-93d3-436f-8475-42710dd08f46.json

```diff
@@ -5,60 +5,81 @@
     "smt_size": 48
   },
   "bios": {
-    "release_date": "12/12/2023",
-    "vendor": "Dell Inc.",
+    "release_date": "2023-12-12",
+    "vendor": "Dell",
     "version": "2.19.0"
   },
   "chassis": {
-    "manufacturer": "Dell Inc.",
-    "name": "PowerEdge R630 (SKU=NotProvided;ModelName=PowerEdge R630)",
+    "manufacturer": "Dell",
+    "name": "PowerEdge R630",
     "serial": "19BKD42"
   },
   "main_memory": {
     "humanized_ram_size": "128 GiB",
-    "ram_size": 134931521536
+    "ram_size": 134929260544
   },
   "monitoring": {
     "wattmeter": false
   },
   "network_adapters": [
     {
+      "device": "eno1",
+      "driver": "bnx2x",
       "enabled": true,
       "mac": "44:a8:42:26:2d:e2",
-      "name": "eno1"
+      "model": "NetXtreme II BCM57800 1/10 Gigabit Ethernet",
+      "rate": 10000000000,
+      "vendor": "Broadcom"
     },
     {
+      "device": "eno2",
+      "driver": "bnx2x",
       "enabled": true,
       "mac": "44:a8:42:26:2d:e4",
-      "name": "eno2"
+      "model": "NetXtreme II BCM57800 1/10 Gigabit Ethernet",
+      "rate": 10000000000,
+      "vendor": "Broadcom"
     },
     {
+      "device": "eno3",
+      "driver": "bnx2x",
       "enabled": false,
       "mac": "44:a8:42:26:2d:e6",
-      "name": "eno3"
+      "model": "NetXtreme II BCM57800 1/10 Gigabit Ethernet",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
     },
     {
+      "device": "eno4",
+      "driver": "bnx2x",
       "enabled": false,
       "mac": "44:a8:42:26:2d:e8",
-      "name": "eno4"
+      "model": "NetXtreme II BCM57800 1/10 Gigabit Ethernet",
+      "rate": 1000000000,
+      "vendor": "Broadcom"
     }
   ],
   "node_name": "chi-02",
   "node_type": "compute_haswell",
   "processor": {
-    "clock_speed": 3100000000,
+    "cache_l1d": 32768,
+    "cache_l1i": 32768,
+    "cache_l2": 262144,
+    "cache_l3": 31457280,
+    "clock_speed": 2411724800,
```

Comment:
Should clock speed better match the description? e.g. 2300000000

Reply:
that's a good question, it depends what clock speed we're talking about.

minimum, maximum non-turbo, max turbo all core, max turbo single core ....
and all of the tools report these differently.

right now, that number is the "current" speed, e.g. whatever that happened to be at the time of measurement.